core/src/util/extrude.cpp

```diff
@@ -1,61 +1,38 @@
 #include "util/extrude.h"
 
-#include "util/floatFormatter.h"
-
+#include "util/yamlUtil.h"
 #include <cmath>
-#include <cstdlib>
 
 namespace Tangram {
 
-Extrude parseExtrudeString(const std::string& _str) {
-
+Extrude parseExtrudeNode(const YAML::Node& node) {
     // Values specified from the stylesheet are assumed to be meters with no unit suffix
     float first = 0, second = 0;
 
-    if (_str == "true") {
-        // "true" means use default properties for both heights, we indicate this with NANs
-        return Extrude(NAN, NAN);
-    }
-
-    if (_str == "false") {
+    bool extrudeBoolean = false;
+    if (YamlUtil::getBool(node, extrudeBoolean)) {
+        if (extrudeBoolean) {
+            // "true" means use default properties for both heights, we indicate this with NANs
+            return Extrude(NAN, NAN);
+        }
         // "false" means perform no extrusion
         return Extrude(0, 0);
     }
 
-    // Parse the first of two possible numbers
-    const char* str = _str.c_str();
-
-    // Get a float if possible & advance pos to the end of the number
-    const int length = _str.length();
-    int count = 0;
-    int offset = 0;
-
-    first = ff::stof(str, length, &count);
-    if (count == 0) {
-        // No numbers found, return zero extrusion
-        return Extrude(0, 0);
+    if (node.IsSequence() && node.size() >= 2) {
```

Comment:
Good to warn if sequence size is >2.

Reply:
Again, erring on the side of simplicity and not warning here.